tests/foreman/api/test_activationkey.py

```diff
@@ -417,6 +427,60 @@ def test_positive_remove_user(self):
         except HTTPError:
             self.fail("Activation Key can't be read")
 
+    @run_in_one_thread
+    @skip_if_not_set('fake_manifest')
+    @tier2
+    def test_positive_fetch_product_content(self):
+        """Associate RH & custom product with AK and fetch AK's product content
+
+        @id: 424f3dfb-0112-464b-b633-e8c9bce6e0f1
+
+        @Assert: Both Red Hat and custom product subscriptions are assigned as
+        Activation Key's product content
+
+        @BZ: 1360239
+
+        @CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        with manifests.clone() as manifest:
+            upload_manifest(org.id, manifest.content)
+        rh_repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+        rh_repo.sync()
+        custom_repo = entities.Repository(
+            product=entities.Product(organization=org).create(),
```

Comment:
@ldjebran red hat products are basically uploaded altogether with subscriptions, if the product was created - then it's definitely a custom one, no need to make emphasis with url.